packages/plugins/rubygem-foreman_discovery/foreman_discovery-15.1.0.gem

```diff
@@ -0,0 +1 @@
+../../../.git/annex/objects/g3/8M/SHA256E-s308736--abf3bc0eb1241a7f80745abe1a955d7bcd82481697068a22a44f80b00c576d04.0.gem/SHA256E-s308736--abf3bc0eb1241a7f80745abe1a955d7bcd82481697068a22a44f80b00c576d04.0.gem
```

Comment:
you didn't delete the old discovery gem, build fails :/

Reply:
Oh thanks. Rebased.